source/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <!-- <meta charset="utf-8"> -->
+    <!-- <meta http-equiv="x-ua-compatible" content="ie=edge"> -->
+    <title>Becky Bosshart</title>
+    <meta name="description" content="The website of Rebecca Bosshart, author, journalist, teacher, redhead.">
+    <!-- <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+
+    <link rel="canonical" href="https://beckybosshart.com/"> -->
```

Comment:
You can leave these commented for now, but you're going to want to uncomment them eventually. It's gonna mess up your styles at first, but ultimately it's going to give you more control over how the site looks on mobile.